doc/flame/examples/lib/rive_example.dart

```diff
@@ -0,0 +1,37 @@
+import 'dart:async';
+
+import 'package:flame/events.dart';
+import 'package:flame/game.dart';
+import 'package:flame_rive/flame_rive.dart';
+
+// ignore: depend_on_referenced_packages
```

Comment:
You shouldn't ignore this, you should add it to the pubspec.

Reply:
fixed!

Reply:
Now the other fix will be merged in a few minutes which will make it possible for you to remove this import completely (we're exporting rive from flame_rive).